app/views/questions/index.html.erb

```diff
@@ -148,5 +166,36 @@
 
     </div>
   </div>
+
+    <!-- Defines how the modal will look -->
+  <div class="modal fade" id="social_display_Modal" role="dialog" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <!-- Sets the style of the modal header  -->
+        <div class="modal-header" style="padding:35px 50px; background-color: #00b3fe; color: #FFFFFF">
```

Comment:
Can we move these extra style parameters into a CSS declaration? You could use a new CSS class that inherits from `modal-header`, or a separate class as well, e.g.:
```
<div class="modal-header-social">
```
(and declare modal-header-social in a stylesheet)